### PR TITLE
feat(info): collapsible info panel

### PR DESCRIPTION
**Pull request by ami**
꒳ᵃ꒳ᵃ꒳ᵃ~(๑°ᗨૢ°๑)♡ӵᵉ੨ᑋ✧

fixes https://github.com/robinjoseph08/pokedextracker.com/issues/17

#### what
- adds collapsible info panel
- uses that collapse functionality for a better responsive/mobile experience

#### notes
still needs:
- [x] detect browser width on load, and collapse info panel if width < 1100px (but not if someone resizes their window to below the threshold, in that case i think it should stay open)
- [x] if the panel is collapsed, when someone clicks the info button on a pokemon, it should open

### CODE REVIEW

File: app/views/info.html [hunk 1]
```diff
@@ -1,10 +1,18 @@
 <div *ngIf="active" class="info-header">
   <img [src]="active.pokemon.icon_url">
-  <h1 [innerHTML]="active.pokemon.name"></h1>
+  <h1></h1>
```
Comment (robinjoseph08) [inline app/views/info.html]: is there a reason you left this empty element here? is it necessary?

**Comment by robinjoseph08** [comment]
wait, am i missing something? this is what it looks like for me:

screen shot 2016-04-03 at 8 57 21 am

**Comment by ami** [comment]
lol I must have messed up the rebase. oopz

**Comment by ami** [comment]
@robinjoseph08 fixed! lols

**Comment by robinjoseph08** [comment]
@ami at what width should it it "auto-collapse" on load? <= 400px?

also, some intermediary sizes are a bit wonky. are you gonna deal with that separately?

screen shot 2016-04-03 at 7 50 44 pm

**Comment by robinjoseph08** [comment]
also the mobile info button hover effects are still there for disabled pokemon. should i create a separate issue or is that a quick fix?

File: app/components/tracker.ts [hunk 2]
```diff
@@ -14,6 +14,8 @@ import { UserService }       from '../services/user';
 
 const HTML = require('../views/tracker.html');
 
+const MOBILE_WIDTH = 750;
```
Comment (ami) [inline app/components/tracker.ts]: this should be 1100 my b

**Pull request merged**